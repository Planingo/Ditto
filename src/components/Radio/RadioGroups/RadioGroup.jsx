import React from 'react';
import { Radio as AntdRadio } from 'antd'


/**
 * 
 * # ðŸš§ _Composant en construction_ ðŸš§
 * 
 * # TODO
 * 
 * ## Description
 * 
 * Documentations andt du composant : https://ant.design/components/radio/
 * 
 */
const RadioGroup = ({ children, ...props }) => {
  return <AntdRadio.Group {...props} />
}

export { RadioGroup };