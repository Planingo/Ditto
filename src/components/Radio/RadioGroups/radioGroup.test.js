import '@testing-library/jest-dom/extend-expect';
import { render } from '@testing-library/react';
import React from 'react';
import {RadioGroup} from './RadioGroup';

describe('RadioGroup component', () => {
  const options = [
    {
      label: 'Apple',
      value: 'Apple',
    },
    {
      label: 'Pear',
      value: 'Pear',
    },
    {
      label: 'Orange',
      value: 'Orange',
    },
  ]
  const radioGroup = (
    <RadioGroup options={options}/>
  );

  it('Snapshot RadioGroup', () => {
    const { baseElement } = render(radioGroup);
    expect(baseElement).toHaveTextContent("Apple");
  });
});