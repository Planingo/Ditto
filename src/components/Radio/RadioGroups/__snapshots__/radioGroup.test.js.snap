// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RadioGroup component Snapshot RadioGroup 1`] = `
<body>
  <div>
    <div
      class="ant-radio-group ant-radio-group-outline css-dev-only-do-not-override-1km3mtt"
    />
  </div>
</body>
`;
